File: Java/Problem_006.java
import java.util.stream.IntStream;

/**
 * 
 * @author Christian Seely
 *
 */
public class Problem_006 {

	/*
	 * Find the sum of the squares of the numbers1 to n.
	 */
	private static int m_SumOfSquares(int n) { return IntStream.range(0,n).map(i -> i*i).sum(); }
	/*
	 * Find the square of the sum of the numbers 1 to n.
	 */
	private static int m_SquareOfSum(int n) { return (int) Math.pow((double) IntStream.range(0,n).sum(),2); }

	public static int problemSix() { return m_SquareOfSum(101)-m_SumOfSquares(101); }

	public static void main(String[] args) { System.out.println(problemSix()); }

}

File: Python/Problem_006.py
# Author: Christian Seely

# Find the sum of the squares of the numbers
# 1 to n.
def sumOfSquares(n):
    return sum([i**2 for i in range(1, n)])

# Find the square of the sum of the numbers
# 1 to n.
def squareOfSum(n):
    return sum(range(1, n)) * sum(range(1, n))

def problemSix():
    return squareOfSum(101)-sumOfSquares(101)

if __name__ == '__main__':
    print (problemSix())